docs/01-introduction.Rmd

```diff
@@ -304,8 +304,8 @@ blogdown::new_site(theme = 'lxndrblz/anatole')
 To save you some time, we list a few themes below that match our taste:
 
 - Simple/minimal themes: [XMin,](https://github.com/yihui/hugo-xmin),
-[Cupper,](https://github.com/zwbetz-gh/cupper-hugo-theme),
-[Tanka,](https://github.com/road2stat/hugo-tanka) [simple-a,](https://github.com/AlexFinn/simple-a) and [ghostwriter.](https://github.com/jbub/ghostwriter)
```

Comment:
@nanxstats Do you want to submit your theme to https://github.com/gohugoio/hugoThemes?

Reply:
@yihui -yes, i will take a look at the submission process over the weekend.

Reply:
Sounds good. I'll put your theme back into the book then. Thanks!

Reply:
Hey @yihui -just a heads up that I found the theme actually requires a larger update to fully work under the current Hugo version, and especially, Goldmark... I should be able to fix them next week after taking care of some work stuff this week. Thanks again for the patience and my apologies for the inconvenience.

Reply:
No worries and no hurry at all. The main goal of this PR was to remove hugo-academic, which has been achieved, so please take your time.